lib/index.js

```diff
@@ -280,7 +280,7 @@ Metalsmith.prototype.readFile = unyield(function*(file){
 
     if (frontmatter && utf8(buffer)) {
       try {
-        parsed = matter(buffer.toString());
+        parsed = matter(buffer.toString().replace(/\r\n/g, '\n'));
```

Comment:
I'm confused. Where in that doc does it state that gray-matter's maintainers are unwilling to support Windows-style line endings?